src/shop.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+  <path d="M21.9982 7.99927V7.02007C22.0006 6.93607 21.991 6.85207 21.9718 6.77047C21.9526 6.68647 21.9238 6.60727 21.8854 6.53527L19.8934 2.55367C19.723 2.21287 19.3774 1.99927 18.9982 1.99927H4.99902C4.61982 1.99927 4.27422 2.21287 4.10382 2.55367L2.11422 6.53527C2.04222 6.67207 1.99902 6.82807 1.99902 6.99367V7.99927C1.99902 9.47527 2.80302 10.7689 3.99822 11.4601V19.9993H2.99982C2.44782 19.9993 1.99902 20.4481 1.99902 21.0001C1.99902 21.5521 2.44782 22.0009 2.99982 22.0009H20.9998C21.5518 22.0009 21.9982 21.5521 21.9982 21.0001C21.9982 20.4481 21.5518 19.9993 20.9998 19.9993H19.999V11.4625C21.1942 10.7713 21.9982 9.47767 21.9982 7.99927ZM5.61582 4.00087H18.3814L19.3798 6.00007H4.61742L5.61582 4.00087ZM13.999 7.99927C13.999 9.10327 13.1014 10.0009 11.9998 10.0009C10.8982 10.0009 9.99822 9.10327 9.99822 7.99927H13.999ZM3.99822 7.99927H7.99902C7.99902 9.10327 7.10142 10.0009 5.99982 10.0009C4.89822 10.0009 3.99822 9.10327 3.99822 7.99927V7.99927ZM9.99822 19.9993V16.9993C9.99822 16.4497 10.447 16.0009 10.999 16.0009H12.9982C13.5502 16.0009 13.999 16.4497 13.999 16.9993V19.9993H9.99822ZM17.9998 19.9993H15.9982V16.9993C15.9982 15.3457 14.6542 13.9993 12.9982 13.9993H10.999C9.34542 13.9993 7.99902 15.3457 7.99902 16.9993V19.9993H5.99982V12.0001C7.19262 12.0001 8.26542 11.4745 8.99982 10.6441C9.73182 11.4745 10.8046 12.0001 11.9998 12.0001C13.195 12.0001 14.2654 11.4745 14.9998 10.6441C15.7318 11.4745 16.8046 12.0001 17.9998 12.0001V19.9993ZM17.9998 10.0009C16.8958 10.0009 15.9982 9.10327 15.9982 7.99927H19.999C19.999 9.10327 19.1014 10.0009 17.9998 10.0009Z" fill="currentColor" stroke-width="0"/>
```

Comment:
Can we pop this in SVGOMG and try to reduce the precision down to 1 decimal place?

Reply:
Done, thanks !